controllers/courses.js

```diff
@@ -147,6 +147,12 @@ const editCourseHandler = (req, res, next) => {
 			time.count = count;
 		});
 
+		// if new course -> add default start and end dates
+		if (!req.params.courseId) {
+			course.startDate = res.locals.currentSchoolData.years.activeYear.startDate;
```

Comment:
pls. use defaultYear instead 
during summer break there is no active year. in that case, the default year will be set to next year by default